docs/features.md

```diff
@@ -32,6 +32,23 @@ How can you target a specific pod by name, if it doesn't have a unique label sel
 
 By default, a disruption affects all containers within the pod. You can restrict the scope of the disruption to a single container or to only some containers [like this](../examples/containers_targeting.yaml).
 
+## Applying a disruption on pod initialization
+
+> :memo: This mode has some restrictions:
+>   * it only works for network related (network and dns) disruptions
+>   * it only works with the pod level
+>   * it does not support containers scoping (applying a disruption to only some containers)
+
+It can be handy to disrupt packets on pod initialization, meaning before containers are actually created and started, to test startup dependencies or init containers. You can do this in only two steps:
+
+* redeploy your pod with the specific label `chaos.datadoghq.com/disrupt-on-init` to hold it in the initialization state
+  * the chaos-controller will inject an init containers named `chaos-handler` in your pod
+  * this init container is lightweight and does nothing but waiting for a `SIGUSR1` signal to complete successfully
+* apply your disruption [with the init mode on](../examples/on_init.yaml)
```

Comment:
It works if the disruption is applied first, yes? As the disruption wont have any targets to pick yet and will wait for the applicable pods to appear